distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -818,9 +818,9 @@ type Vector a
          The following code returns 2.
 
              [1, 2, 3, 4].second
-    second : Vector ! Singleton_Error
+    second : Vector ! Index_Out_Of_Bounds_Error
     second self = if self.length >= 2 then self.unsafe_at 1 else
-        Error.throw (Singleton_Error_Data self)
+        Error.throw (Index_Out_Of_Bounds_Error_Data 1 self.length)
```

Comment:
Cannot we just delegate to `self.at 1` and rely on it to throw the right IOOB error?

Reply:
:)